apps/storybook/stories/ZIndex.stories.mdx

```diff
@@ -0,0 +1,30 @@
+import { Meta } from "@storybook/addon-docs";
+import ZIndex from "./ZIndex";
+
+<Meta title="z-Index" />
```

Comment:
autodocs unfortunately dont work here because it's a function and not an actual component. so i manually created docs on how to use this function